plugins/woocommerce/src/Blocks/Domain/Services/CheckoutFieldsFrontend.php

```diff
@@ -217,10 +227,11 @@ public function save_account_form_fields( $user_id ) {
 		$location_validation = $this->checkout_fields_controller->validate_fields_for_location( $field_values, 'contact', 'other' );
 
 		if ( is_wp_error( $location_validation ) && $location_validation->has_errors() ) {
-			wc_add_notice( $location_validation->get_error_message(), 'error' );
+			foreach ( $location_validation->get_error_messages() as $error_message ) {
+				wc_add_notice( $error_message, 'error' );
```

Comment:
Didn't quite figure this one out, do we still want to save customer data after this?

Reply:
If there's a validation error, then no